src/OpenTelemetry.Exporter.Zipkin/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Support Activity `Status` and `StatusDescription` in Zipkin Exporter.
```

Comment:
Added support for Activity `Status` and `StatusDescription` which were added to Activity from version 6.0. To maintain backward compatibility, the exporter falls back to checking status inside the special tags "otel.status_code", "otel.status_description", if Status is not set using the new Status field